torch/autograd/profiler.py

```diff
@@ -486,6 +486,70 @@ def _call_end_callbacks_on_future(self, fut: Future[Any]) -> Future[Any]:
         return profiled_future
 
 
+class emit_itt(object):
```

Comment:
Nit: `EmitITT`. `profile` is cased like a function for historic reasons, but new code doesn't need to.

Reply:
I don't follow here. Should we change `emit_itt` to another word? This aligns with `emit_nvtx`.